belt-block/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "belt-block"
+version = "0.1.0"
+description = "betl-block block cipher implementation"
+authors = ["RustCrypto Developers", "Alexandr Kitaev"]
+license = "MIT OR Apache-2.0"
+rust-version = "1.56"
+edition = "2021"
+readme = "README.md"
+documentation = "https://docs.rs/belt-block
+repository = "https://github.com/RustCrypto/block-ciphers"
+keywords = ["crypto", "belt-block", "belt", "stb"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Remove this comment.

Reply:
The comment is still present.

Reply:
Sorry, haven't pushed it. Now it's done.